Dockerfile.microos.dapper

```diff
@@ -0,0 +1,15 @@
+ARG TUMBLEWEED=opensuse/tumbleweed
```

Comment:
Is this argument actually used anywhere/for debugging or are we just arbitrarily defining it?

Reply:
I do this with all of my Dockerfiles. It lets me swap in pinned versions, or otherwise compatible images, on the CLI or via Makefile which I somehow always end up needing. I hate checking in accidental cruft and/or the git mess that changing files like this for testing/assertion can cause.

---

Yes, I am weird about this.